pkg/fuzzer/job.go

```diff
@@ -437,12 +437,12 @@ type hintsJob struct {
 }
 
 func (job *hintsJob) run(fuzzer *Fuzzer) {
-	// First execute the original program twice to get comparisons from KCOV.
-	// The second execution lets us filter out flaky values, which seem to constitute ~30-40%.
+	// First execute the original program several times to get comparisons from KCOV.
+	// Additional executions lets us filter out flaky values, which seem to constitute ~30-40%.
 	p := job.p
 
 	var comps prog.CompMap
-	for i := 0; i < 2; i++ {
+	for i := 0; i < 3; i++ {
```

Comment:
I wonder what happens if we fail to execute the request -- e.g. get some errno and zero (or near zero) comparison arguments. Once it happens, we basically discard all data?

With 3 iteration, it becomes even more likely than with 2. We should somehow protect against this.

Reply:
Done. Changed return on unsuccessful runs to continue.